stingray/deadtime/fad.py

```diff
@@ -316,6 +282,110 @@ def calculate_FAD_correction(lc1, lc2, segment_size, norm="none", gti=None,
     return results
 
 
+def calculate_FAD_correction(lc1, lc2, segment_size, norm="none", gti=None,
+                             plot=False, ax=None, smoothing_alg='gauss',
+                             smoothing_length=None, verbose=False,
+                             tolerance=0.05, strict=False,
+                             output_file=None, return_objects=False):
+    """Calculate Frequency Amplitude Difference-corrected (cross)power spectra.
+
+    Reference: Bachetti \& Huppenkothen, 2018, ApJ, 853L, 21
```

Comment:
Oh, so it's this one. But why is this one and `FAD` separate functions? Why not just make this into one?

Reply:
This is the old function, with exactly the same interface as before. I didn't want to introduce a breaking change, and I also liked a short name better. New users will read in the docs to use the new one, that also accepts event lists and is more general, but old users will not be damaged.

Reply:
I should maybe deprecate this?

Reply:
Just because it is an acronym and I wanted the function name to mean the acronym. "fad" as a name means "craze". Probably overthinking it, but that's why